pkg/base/base.go

```diff
@@ -1,11 +1,56 @@
 package base
 
+import (
+	"github.com/wttech/aemc/pkg/cfg"
+)
+
 type Opts struct {
-	TmpDir string
+	TmpDir           string
+	ChecksumExcludes []string
 }
 
 func NewOpts() *Opts {
 	return &Opts{
 		TmpDir: "aem/home/tmp",
+		ChecksumExcludes: []string{
+			// AEM files
+			"**/aem/home/**",
+
+			// meta files
+			"**/.*",
+			"**/.*/**",
+			"!.content.xml",
```

Comment:
does AEM have any more files with such strange name? I mean dot in the beginning?